emcc.py

```diff
@@ -1430,6 +1430,7 @@ def has_c_source(args):
       shared.Settings.EXPORTED_FUNCTIONS += ['_malloc', '_free']
 
     if shared.Settings.WASM_BACKEND:
+      shared.Settings.EXPORTED_FUNCTIONS += ['_stackSave', '_stackRestore', '_stackAlloc']
```

Comment:
I thought these were not prefixed?

Reply:
Yeah its a little confusing.   At this point, these are now just normal native symbols, just like C symbols.  I think the distinction must only really matter for fastcomp because all the tests pass ...

Reply:
Another way of putting it:

With fastcomp we have the C symbols which start with "_" and then we can have other "native" asm.js symbols that don't have the "_".   So there are two different styles for asmjs native symbols.

With the backend we never have "_" on any of our native symbols, so there is only one style.  But for backwards compat we expect the "_" prefix on native symbols in `EXPORTED_FUNCTIONS`.

Does that help?

Reply:
Yeah, sounds fine to me, assuming the compiler likes it too :wink: